pkg/driver/camera/camera_linux.go

```diff
@@ -23,8 +23,13 @@ const (
 )
 
 var (
-	errReadTimeout = errors.New("read timeout")
-	errEmptyFrame  = errors.New("empty frame")
+	errReadTimeout       = errors.New("read timeout")
+	errEmptyFrame        = errors.New("empty frame")
+	supportedResolutions = [][2]int{
+		{1920, 1080}, // 1080p
+		{1280, 720},  // 720p
+		{640, 480},   // 480p
```

Comment:
Maybe better to have https://commons.wikimedia.org/wiki/File:Vector_Video_Standards2.svg these sizes?

Reply:
Yep, makes sense. Updated with these resolutions.